flink-doris-connector/src/main/java/org/apache/doris/flink/serialization/RowBatch.java

```diff
@@ -90,10 +89,12 @@ public void put(Object o) {
     private List<FieldVector> fieldVectors;
     private RootAllocator rootAllocator;
     private final Schema schema;
-    private final String DATETIME_PATTERN = "yyyy-MM-dd HH:mm:ss";
-    private final String DATETIMEV2_PATTERN = "yyyy-MM-dd HH:mm:ss.SSSSSS";
-    private final DateTimeFormatter dateTimeFormatter = DateTimeFormatter.ofPattern(DATETIME_PATTERN);
-    private final DateTimeFormatter dateTimeV2Formatter = DateTimeFormatter.ofPattern(DATETIMEV2_PATTERN);
+    private final String datetimePattern = "yyyy-MM-dd HH:mm:ss";
+    private final String datetimeV2Pattern = "yyyy-MM-dd HH:mm:ss.SSSSSS";
```

Comment:
Change to uppercase,   can change it to static

Reply:
Yes, changed!